svf/include/Graphs/ICFG.h

```diff
@@ -148,27 +152,27 @@ class ICFG : public GenericICFGTy
     }
 
 protected:
-    /// Remove a SVFG edge
+    /// Add control-flow edges for top level pointers
```

Comment:
Change this comment to:

Add intraprocedural and interprocedural control-flow edges.